src/kdump-tools/patch/0003-Panic-on-kdump-kernel.patch

```diff
@@ -0,0 +1,20 @@
+From: Olivier Singla <olivier.singla@broadcom.com>
+Date: Mon Dec  2 15:25:46 EST 2019
+Subject: kdump: add "panic=X" option so system reboots if the capture kernel crashes or stay stuck
+
+In case the capture kernel (the kdump kernel started with kexec) would
+either crash or be stuck, the system should reboot. We need then to add
+the "panic=X" option to the kernel. Without this option, the system could
+stuck and not reboot.
```

Comment:
the patch description is not complete. you have added a few other options, such as debug, hpet, pcie_port, pci, PLATFORM. can you describe those purpose as well?

Reply:
@lguohan 

After taking a second look at these two proposed patches, I feel we should not add patches to kdump-tools package. Instead these additional configurations need to be appended to /etc/default/kdump-config as part of build_debian.sh. This will make the SONiC kdump customizations easier to manage. I will also add appropriate description for the options added.

Comments?

Reply:
i agree with you. okay to move to build_debian.sh.

Reply:
Agree with you are renamed it as sonic_platform.